yt_dlp/extractor/manoto.py

```diff
@@ -0,0 +1,102 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+
+
+class ManotoTVIE(InfoExtractor):
+    IE_NAME = 'Manoto TV (Episode / Video)'
+    _VALID_URL = r'https?://(?:www\.)?manototv\.com/episode/(?P<id>[0-9]+)'
```

Comment:
the site doesn't have individual videos?
If it does then add to tests if it doesn't then remove ` / Video` from IE name

Reply:
Makes sense.